### ISSUE TITLE
Feel++

### ISSUE THREAD

**Issue by vhuber** [kostyfisik/FEA-compare#4]
have you considered [Feel++](github.com/feelpp/feelpp) ?

**Comment by kostyfisik** [comment]
No, but it would be great if you can provide a PR with a profile for Feel++ :) This is the reason why I am trying to provide a good comparison in a single place - there are so many packages, it is really hard just to find them all...

**Comment by vhuber** [comment]
Ok, I've forked and starting working on a profile.
I guess I only have to fill a `profiles/feelpp.profile` copied from any other ?

**Comment by kostyfisik** [comment]
https://github.com/kostyfisik/FEA-compare/blob/master/main-keys.txt is used for key sorting, so  you can use it as an empty template. For sure you can use any available profile as a starting point too.

**Comment by vhuber** [comment]
I haven't seen a clone has already be done in feelpp. A PR will be submitted within the next two weeks.

**Comment by kostyfisik** [comment]
@vhuber Any update? :)
